### DISCUSSION TITLE
빌드잇 모델링 데이터 재사용 관련

### DISCUSSION THREAD

**Discussion by ellejou76** [naverz/zepeto-studio-kor#737] [5. Q&A]
안녕하세요.

커스텀 월드를 제작 중인데,  빌드잇에서 제공되는 기본 모델링 소스들을 가져다  모양을 수정 변형해서 재사용하는 것도 괜찮을까요?
혹시, 저작권이나. 제페토 정책에 위반되는 사항일까요?

감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 재사용 오브젝트가 가이드라인을 위반하지 않는다면 가능합니다.
아래 가이드 라인 참고 부탁 드립니다. 
https://github.com/naverz/zepeto-studio-kor/discussions/595
